packages/dev/core/src/Physics/physicsShapeProximityQuery.ts

```diff
@@ -0,0 +1,33 @@
+import type { Quaternion, Vector3 } from "../Maths/math.vector";
+import type { PhysicsShape } from "./v2/physicsShape";
+import type { PhysicsBody } from "./v2/physicsBody";
+
+/**
+ * Query for shape proximity.
+ */
+export interface IPhysicsShapeProximityQuery {

```

Comment:
I would name it IPhysicsProximityCastQuery instead, to make the difference with IPhysicsShapeCastQuery more obvious